Formula/docker-machine-driver-hyperkit.rb

```diff
@@ -0,0 +1,35 @@
+class DockerMachineDriverHyperkit < Formula
+  desc "Docker Machine driver for hyperkit"
+  homepage "https://github.com/machine-drivers/docker-machine-driver-hyperkit"
+  url "https://github.com/machine-drivers/docker-machine-driver-hyperkit.git",
+    :tag => "v1.0.0",
+    :revision => "88bae774eacefa283ef549f6ea6bc202d97ca07a"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+  depends_on :macos => :yosemite
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/machine-drivers/docker-machine-driver-hyperkit").install buildpath.children
+    cd "src/github.com/machine-drivers/docker-machine-driver-hyperkit" do
+      system "dep", "ensure"
+      system "go", "build", "-o", "#{bin}/docker-machine-driver-hyperkit",
+             "-ldflags", "-X main.version=#{version}"
+      prefix.install_metafiles
+    end
+  end
+
+  def caveats; <<~EOS
+    This driver requires superuser privileges to access the hypervisor. To
+    enable, execute
+        sudo chown root:wheel #{opt_prefix}/bin/docker-machine-driver-hyperkit
```

Comment:
`opt_bin`. (I appreciate the other formula to do this has the wrong style used as well).

Reply:
Nice, fixed 👍